<?xml version="1.0"?>
<robot name="edumip_my_robot">

  <material name="blue">
    <color rgba="0 0 0.8 1"/>
  </material>

  <material name="black">
    <color rgba="0 0 0 1"/>
  </material>

  <link name="edumip_body">
    <visual>
      <origin xyz="0 0 0.05" rpy="0 -0.2 0"/>
      <geometry>
        <box size="0.035 0.05 0.1"/>
      </geometry>
      <material name="blue"/>
    </visual>
  </link>

  <link name="wheelL">
    <visual>
      <origin rpy="1.57075 0 0" xyz="0 0 0"/>
      <geometry>
        <cylinder length="0.01" radius="0.034"/>
      </geometry>
      <material name="black"/>
    </visual>
  </link>

  <joint name="jointL" type="continuous">
    <axis xyz="0 1 0" rpy="0 0 0" />
    <parent link="edumip_body"/>
    <child link="wheelL"/>
    <origin rpy="0 0 0" xyz="-0.025 0.035 0.0"/>
  </joint>

  <link name="wheelR">
    <visual>
      <origin rpy="1.57075 0 0" xyz="0 0 0"/>
      <geometry>
        <cylinder length="0.01" radius="0.034"/>
      </geometry>
      <material name="black"/>
    </visual>
  </link>

  <joint name="jointR" type="continuous">
    <axis xyz="0 1 0" rpy="0 0 0" />
    <parent link="edumip_body"/>
    <child link="wheelR"/>
    <origin rpy="0 0 0" xyz="-0.025 -0.035 0.0"/>
  </joint>

</robot>
